graphstorm-processing/graphstorm_processing/data_transformations/dist_transformations/dist_numerical_transformation.py

```diff
@@ -209,8 +237,166 @@ def gauss_transform(rank: pd.Series) -> pd.Series:
         scaled_df = normalized_df.orderBy(original_order_col).drop(
             value_rank_col, original_order_col
         )
+    else:
+        raise ValueError(f"Unsupported normalization requested: {shared_norm}")
+
+    return NormalizationResult(scaled_df, norm_representation)
 
-    return scaled_df
+
+def _apply_standard_transform(
+    input_df: DataFrame,
+    cols: list[str],
+    out_dtype: str,
+    col_sums: Optional[dict[str, float]] = None,
+) -> tuple[DataFrame, dict]:
+    """Applies standard scaling to the input DataFrame, individually to each of the columns.
+
```

Comment:
Can you add an equation of standard transformation?

Reply:
Is this transformation supported in gconstuct?
If not, we should implement it for gconstruct.

Reply:
Will add an explanation. This is a transformation ported from NeptuneML, each value in a column is divided by the sum of the column.

If we want to implement that for GConstruct should be straightforward, not sure if we want it for 0.4?

Reply:
If it is straightforward, let's do it.

Reply:
I'll a P1 task for v0.4